docs/book/src/clusterctl/commands/generate-yaml.md

```diff
@@ -22,6 +22,16 @@ clusterctl generate yaml --from https://github.com/foo-org/foo-repository/blob/m
 # Generates a configuration file with variable values using
 # a template stored locally.
 clusterctl generate yaml  --from ~/workspace/cluster-template.yaml
+
+# Prints list of variables used in the local template
+clusterctl generate yaml --from ~/workspace/cluster-template.yaml --list-variables
+
+# Prints list of variables from template passed in via stdin
+cat ~/workspace/cluster-template.yaml | clusterctl generate yaml --from - --list-variables
```

Comment:
```suggestion
cat ~/workspace/cluster-template.yaml | clusterctl generate yaml --list-variables --from - 
```

Reply:
Did we make the `--from=-` the default? If so, we could probably drop it from the examples

Reply:
Yeah...I documented the default behavior on the next line but also wanted to explicitly show the flag input for stdin. I thought I'd be more explicit here since these are the docs.

I removed the `--from -` from the CLI help context to `cat ~/workspace/cluster-template.yaml | clusterctl generate yaml --list-variables`